cmd/vspadmin/README.md

```diff
@@ -0,0 +1,30 @@
+# vspadmin
+
+vspadmin is a tool to perform various VSP administration tasks.
+
+## Usage
+
+```no-highlight
+vspadmin [OPTIONS] COMMAND
+```
+
+## Options
+
+```no-highlight
+--homedir=                         Path to application home directory. (default: /home/user/.vspd)
+--network=[mainnet|testnet|simnet] Decred network to use. (default: mainnet)
+-h, --help                         Show help message
+```
+
+## Commands
+
+### `createdatabase`
+
+Creates a new database for a new deployment of vspd. Accepts the xpub key to be
+used for collecting fees as a parameter.
+
+Example:
+
+```no-highlight
+  go run ./cmd/vspadmin createdatabase <xpub>
```

Comment:
I personally think `go install` with a path to the repo is much preferable over run like this.  First, you have to clone the repo to be able to do the `go run` like this.  Second, you have to build the bin every time with `go run`.

On the other hand, `go install github.com/decred/vspd/cmd/vspadmin@latest` works without cloning and installs the binary.

The same applies to the other cases too, but I'll just comment on this one.